schedule/functional/extra_tests_gnome.yaml

```diff
@@ -3,7 +3,20 @@ description:    >
     Maintainer: asmorodskyi, okurz.
     Extra tests for software in desktop applications which were designed to run on gnome
     VNC_STALL_THRESHOLD is needed for xen svirt to don't turn off the scrreen after default 4 sec
+conditional_schedule:
```

Comment:
JFYI, we have created `installation/bootloader_start` as an abstraction, so that in schedule you don't really care what will be called, but the system will be spawned. We have preferred this solution to conditional schedule which has to be added to each schedule file.

Reply:
I will use `installation/bootloader_start` after I get all our extra_tests* on YAML schedule, ATM I just need quick merges before the next build. And by just replicating the current main.pm schedule, I can verify with just `_EXIT_AFTER_SCHEDULE=1`.

Reply:
@SergioAtSUSE make sure we have a ticket for it.. :) just to not forget to do the switch